index.html

```diff
@@ -324,6 +319,51 @@
 might be best suited to a zero-knowledge system
 that provides strong unlinkability.
 
+### Use Case: Email Addresses
```

Comment:
As with phone numbers, I'm not sure emails are part of OpenID4VP or ISO18013-5 mandate those... tho they could be asked for, because you can have random fields in there.